sys/include/net/ipv6/addr.h

```diff
@@ -729,6 +729,18 @@ char *ipv6_addr_to_str(char *result, const ipv6_addr_t *addr, uint8_t result_len
  */
 ipv6_addr_t *ipv6_addr_from_str(ipv6_addr_t *result, const char *addr);
 
+/**
+ * @brief split IPv6 address string representation and return remaining string
+ *
+ * @note Will change @p separator position in @p addr_str to '\0'
```

Comment:
This is more a detail than a note
```suggestion
 * Will change @p separator position in @p addr_str to '\0'
```